### ISSUE TITLE
Won't run.  .NET is installed, but not recognized

### ISSUE THREAD

**Issue by teknowledgist** [TheJoeFin/Text-Grab#235]
**Describe the bug**
I'm getting a message to install/update .NET when attempting to run Text-Grab.

**Where is the bug**
![image](https://user-images.githubusercontent.com/3977744/211348279-15d7a7d7-e88a-4955-95a5-902690cb473b.png)

To be fair, the error says it needs v6.0.0, but clicking "yes" opens a download page for 6.0.12.  I don't think 6.0.0 is available.

**To Reproduce**
Install .NET Runtime, double-click on Text-Grab

**Expected behavior**
Text-Grab starts

**Where did you get Text Grab?**
- GitHub releases

**Desktop (please complete the following information):**
 - OS:  Windows 10
 - Version: 21H2
 - Text Grab Version: 3.1.1

**Comment by TheJoeFin** [comment]
Hello @teknowledgist! I tried to replicate this bug but was unsuccessful. I uninstalled my .NET installations and downloaded the latest and Text Grab worked fine. Maybe having more than just .NET6 installed is confusing it? 

In the releases for Text Grab there is a .ZIP with -SC on the end. This is for "Self Contained" and should work even without .NET installed on the system. Let me know if that works for you.

**Comment by teknowledgist** [comment]
The SC version works fine, but I don't know what to say about the other.

I reset the VM I was using to a minimal "fresh" setup and installed only the .NET runtime v6.0.13, and I got the same error.  I also tried running the .exe on my everyday, production machine and get the same result.  The latter system is in a domain with some security GPOs, but the VM is stand-alone and uninhibited (other than some external firewall settings).
